packages/react/src/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+interface TabProps {
+  value: number;
+  index: number;
+  label?: string;
+  children?: React.ReactNode;
+}
+
+const Tab = ({ value, label, index, children }: TabProps) => {
+  const tabRef = useRef<HTMLLIElement>(null);
+  const tabIndex = index === value ? 0 : -1;
+  const selected = tabIndex === 0;
+  if (selected) {
+    tabRef?.current?.focus();
+  }
+
+  return (
+    <li
+      role="tab"
+      className={classNames('Tab', {
+        'Tab--active': selected
+      })}
+      id={`tab-${index}`}
+      aria-controls={`tabpanel-${index}`}
+      tabIndex={tabIndex}
+      aria-selected={selected}
+      ref={tabRef}
+    >
+      {label || children}
```

Comment:
What's the point of even having a special `label` prop here?

Is `<Tab label="Foo" />` better/easier/etc than `<Tab>Foo</Tab>`?